composer.json

```diff
@@ -12,7 +12,7 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "symfony/yaml": "~2.0"
+        "symfony/yaml": "~2.0|~3.0"
```

Comment:
I suggest forbidding the 2.0 versions as the autoloading config is broken in them, which would break the ``prefer-lowest`` build